band/java/src/main/native/native_buffer_wrapper_jni.cc

```diff
@@ -0,0 +1,60 @@
+#include <jni.h>
+
+#include "band/buffer/buffer.h"
+#include "band/java/src/main/native/jni_utils.h"
+
+/*
+
+
+  private native long createBuffer();
+
+  private native void deleteBuffer(long bufferHandle);
+
+  private native void setFromByteBuffer(
+      long bufferHandle, final byte[] buffer, int width, int height, int
+  bufferFormat);
+
+  private native void setFromYUVBuffer(long bufferHandle, final byte[] y, final
+  byte[] u, final byte[] v, int width, int height, int yRowStride, int
+  uvRowStride, int uvPixelStride);
+
+*/
+
+using band::Buffer;
+
+extern "C" {
+
+JNIEXPORT void JNICALL Java_org_mrsnu_band_Buffer_deleteBuffer(
```

Comment:
These native functions should be named `Java_org_mrsnu_band_NativeBufferWrapper_mtd`.